### PR TITLE
[Merged by Bors] - chore: fix docstring of `HeytingAlgebra.himp_bot`

### PR DESCRIPTION
**Pull request by Seasawher**
fix docstring of `HeytingAlgebra.himp_bot`

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Heyting/Basic.lean [hunk 1]
```diff
@@ -142,7 +142,7 @@ class GeneralizedCoheytingAlgebra (α : Type*) extends Lattice α, OrderBot α,
 /-- A Heyting algebra is a bounded lattice with an additional binary operation `⇨` called Heyting
 implication such that `a ⇨` is right adjoint to `a ⊓`. -/
```
Comment (urkud) [inline Mathlib/Order/Heyting/Basic.lean:143]: Could you please use the proper Lean 4 notation here?
```suggestion
implication such that `(a ⇨ ·)` is right adjoint to `(a ⊓ ·)`. -/
```

**Comment by urkud** [comment]
Thanks!
bors d+

File: Mathlib/Order/Heyting/Basic.lean (hunk 1 above)
Comment (Seasawher) [reply Mathlib/Order/Heyting/Basic.lean:143]: Same modification is needed here?

```lean
class GeneralizedHeytingAlgebra (α : Type*) extends Lattice α, OrderTop α, HImp α where
  /-- `a ⇨` is right adjoint to `a ⊓` -/
  le_himp_iff (a b c : α) : a ≤ b ⇨ c ↔ a ⊓ b ≤ c
```

Comment (YaelDillies) [reply Mathlib/Order/Heyting/Basic.lean:143]: Yes indeed

Comment (Seasawher) [reply Mathlib/Order/Heyting/Basic.lean]: done!

File: Mathlib/Order/Heyting/Basic.lean [hunk 2]
```diff
@@ -124,36 +124,36 @@ theorem hnot_apply [∀ i, HNot (π i)] (a : ∀ i, π i) (i : ι) : (￢a) i =
 end Pi
 
 /-- A generalized Heyting algebra is a lattice with an additional binary operation `⇨` called
-Heyting implication such that `a ⇨` is right adjoint to `a ⊓`.
+Heyting implication such that `a ⇨ ·` is right adjoint to `a ⊓ ·`.
```
Comment (urkud) [inline Mathlib/Order/Heyting/Basic.lean:127]: Lean 4 expects parentheses around lambda functions written with `·` (here and elsewhere).
```suggestion
Heyting implication such that `(a ⇨ ·)` is right adjoint to `(a ⊓ ·)`.
```

Comment (Seasawher) [reply Mathlib/Order/Heyting/Basic.lean]: done!

**Review by YaelDillies — APPROVED** [review]
maintainer merge

**Comment by urkud** [comment]
@YaelDillies It's already delegated.

**Comment by b-mehta** [comment]
bors merge

**Pull request closed without merging**